.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,25 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit codespell darker flake8 flake8-2020 flake8-bugbear
+                         flake8-comprehensions isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
+      - run: darker --check .
+      - run: codespell || true  # --ignore-words-list="" --skip="*.css,*.js,*.lock"
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88
+                      --show-source --statistics
+      - run: isort --check-only --profile black . || true
+      - run: pip install -r requirements.txt || pip install --editable . || true
+      - run: mkdir --parents --verbose .mypy_cache
+      - run: mypy --ignore-missing-imports --install-types --non-interactive . || true
```

Comment:
`mypy` is already being run using [pytest-darker](https://pypi.org/project/pytest-darker/) as well, see
- [`pytest.ini`](https://github.com/akaihola/darker/blob/8a1835c952b9743ea5cb6d40d23eb9c6458e7828/pytest.ini#L8)
- [`mypy.ini`](https://github.com/akaihola/darker/blob/8a1835c952b9743ea5cb6d40d23eb9c6458e7828/mypy.ini)

And again, if we'd run `mypy` explicitly instead of through [pytest-darker](https://pypi.org/project/pytest-darker/), `--mypy` needs to be removed from [`pytest.ini`](https://github.com/akaihola/darker/blob/8a1835c952b9743ea5cb6d40d23eb9c6458e7828/pytest.ini#L8). At least the `--ignore-missing-imports` option should be enabled per package in [`mypy.ini`](https://github.com/akaihola/darker/blob/8a1835c952b9743ea5cb6d40d23eb9c6458e7828/mypy.ini) (as it's already being done) instead of using the command line options. This way running `mypy` or `darker --mypy` manually would give the same results as the CI run.

Reply:
error: --non-interactive is only supported with --install-types

Reply:
> error: --non-interactive is only supported with --install-types

Thanks, corrected.